Index.html

```diff
@@ -0,0 +1,18 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```

Comment:
Your using indentation of four spaces here but on other lines (line 11 for example) you're using two. I would pick on or the other.

Reply:
I corrected the indentation 

<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Weather</title>
  <link rel="stylesheet" href="style.css">
</head>
<body>